server/v2/cometbft/flags/flags.go

```diff
@@ -0,0 +1,79 @@
+package flags
+
+import "github.com/spf13/cobra"
+
+const (
+	FlagQuery   = "query"
+	FlagType    = "type"
+	FlagOrderBy = "order_by"
+)
+
+const (
```

Comment:
do we still need all these cc @julienrbrt

Reply:
We should only have the one we use here for sure. The rest seems duplicated.